docs/architecture/adr-061-liquid-staking.md

```diff
@@ -0,0 +1,88 @@
+
+A new governance parameter is introduced that defines the ratio of exempt to issued tokenized shares. This is called the exemption factor.
+
+Min self delegation is removed from the staking system with the expectation that it will be replaced by the exempt delegations sytem. The exempt delegation system allows multiple accounts to demonstrate economic alignment with the validator operator as team members, partners etc without co-mingling funds. The exempt delegation will likely be required to grow the validators business under widespread adoption of liquid staking and where governence sets the exemption factor.
+
+When shares are tokenized, the underlying shares are transfered to a module account and rewards go to the module account for the TokenizedShareRecord. 
+
+There is no longer a mechanism to override the validators vote for TokenizedShares.
+
+
+### `MsgTokenizeShares`
+
+The MsgTokenizeShares message is used to create tokenize delegated tokens. This message can be executed by any delegator who has positive amount of delegation and after execution the specific amount of delegation disappear from the account and share tokens are provided. Share tokens are demoninated in the validator and record id of the underlying delegation.
+
+A user may tokenize some or all of their Delegation.
+
+They will recieved shares with the denom of `cosmosvaloper1xxxx/5` where 5 is the record id for the validator operator.
+
+MsgTokenizeShares fails if the account is a VestingAccount. Users will have to move vested tokens to a new account and endure the unbonding period. We view this as acceptable tradeoff vs complex book keeping of vested tokens.
+
+The total amount of outstanding tokenized shares for the validator is checked against the sum of exempt delegation multiplied by the exemption factor. If the tokenized shares would exceed this limit, the execution fails.
+
+MsgTokenizeSharesResponse provides the number of tokens generated and their denom.
+
+
+### `MsgRedeemTokensforShares`
+
+The MsgRedeemTokensforShares message is used to redeem the delegation from share tokens. This message can be executed by any user who owns share tokens and after execution the delegation appear for the user.
+
+### `MsgTransferTokenizeShareRecord`
+
+The MsgTransferTokenizeShareRecord message is used to transfer the ownership of rewards generated from the tokenized amount of delegation. The tokenize share record is created when a user tokenize his/her delegation and deleted when the full amount of share tokens are redeemed.
+
+This is designed to work with liquid staking designs that do not redeem the tokenized shares and may instead want to keep the shares tokenized.
+
+
+### `MsgExemptDelegation`
+
+The MsgExemptDelegation message is used to exempt a delegation to a validator. If the exemption factor is greater than 0, this will enable more delegation to the validator. 
```

Comment:
```suggestion
The MsgExemptDelegation message is used to exempt a delegation to a validator. If the exemption factor is greater than 0, this will allow more delegation shares to be issued from the validator.
```